package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-test-runner-qti",
-    "version": "2.13.6",
+    "version": "2.13.7",
```

This is definitely a new feature, not a fix.
Consider bumping a minor version instead.
```suggestion
    "version": "2.14.0",
```